geograypher/meshes/meshes.py

```diff
@@ -123,6 +123,16 @@ def __init__(
         )
         # Load the texture
         self.logger.info("Loading texture")
+        # load IDs_to_labels
+        # if IDs_to_labels not provided, check the directory of the mesh and get the file if found
+        if IDs_to_labels == None and isinstance(mesh, PATH_TYPE.__args__):
+            for file in mesh.parent.iterdir():
+                if 'IDs_to_labels.json' in file.name:
+                    IDs_to_labels = file.name
+        # convert IDs_to_labels from file to dict
+        if isinstance(IDs_to_labels, PATH_TYPE.__args__):
```

Comment:
I assume the `__args__` makes this a valid typecheck? I only tried `PATH_TYPE`, got an error, and didn't look any further. Nice work.

Reply:
Correct.